### PR TITLE
Add external contribution documentation

### PR DESCRIPTION
**Pull request by laurenfrederick**
**Summary:** Summary of changes

Addresses [CUMULUS-XX: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-XXX)

## Changes

* Detailed list or prose of changes
* ...

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Adhoc testing
- [ ] Integration tests

### CODE REVIEW

File: docs/external-contributions/external-contributions.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+id: external-contributions
+title: External Contributions
+hide_title: true
```
Comment (markdboyd) [inline docs/external-contributions/external-contributions.md:4]: Nitpick: In conjunction with #1887, can we do `hide_title: false` here?

File: docs/external-contributions/external-contributions.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+---
+
+# External Contributions
+
+Contributions to Cumulus may be made in the form of [PRs to the repositories directly](https://github.com/nasa/cumulus/blob/master/CONTRIBUTING.md) or through externally developed tasks and components. Cumulus is designed as an ecosystem that leverages Terraform deployments and AWS Step Functions to easily integrate external components.
+
+This list may not be exhaustive and represents components are open source and owned externally that have been tested with the Cumulus system. For more information and contributing guidelines, visit the respective GitHub repositories.
```
Comment (markdboyd) [inline docs/external-contributions/external-contributions.md:11]:
```suggestion
This list may not be exhaustive and represents components that are open source and owned externally and that have been tested with the Cumulus system. For more information and contributing guidelines, visit the respective GitHub repositories.
```

Just a thought: do we list all Cumulus repositories anywhere? Should we?

File: docs/external-contributions/external-contributions.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+## Distribution
+
+[The ASF Thin Egress App](https://github.com/asfadmin/thin-egress-app#welcome-to-tea---the-thin-egress-app) is a key component of Cumulus distribution. TEA can be deployed [with Cumulus](../deployment/thin_egress_app) or as part of other applications distribution data.
```
Comment (markdboyd) [inline docs/external-contributions/external-contributions.md:15]:
```suggestion
[The ASF Thin Egress App](https://github.com/asfadmin/thin-egress-app#welcome-to-tea---the-thin-egress-app) is used by Cumulus for distribution. TEA can be deployed [with Cumulus](../deployment/thin_egress_app) or as part of other applications to distribute data.
```

File: docs/external-contributions/external-contributions.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+### CNM
+
+PO.DAAC provides two workflow tasks to be used with the [Cloud Notification Mechanism (CNM) Schema](https://github.com/podaac/cloud-notification-message-schema#cumulus-sns-schema): [CNM to Granule](https://github.com/podaac/cumulus-cnm-to-granule#cnm-to-granule-task) and [CNM Response](https://github.com/podaac/cumulus-cnm-response-task#cnm-response-task).
+
+See the [CNM workflow data cookbook](../data-cookbooks/cnm-workflow) for an example of how these can be used in a Cumulus ingest workflow.
+
+### DMR++ Generation
+
+GHRC has provided a [DMR++ Generation](https://github.com/ghrcdaac/dmrpp-generator#overview) wokrflow task. This is meant to be used in conjunction with Cumulus' [Hyrax Metadata Updates workflow task](https://github.com/nasa/cumulus/tree/master/tasks/hyrax-metadata-updates#cumulushyrax-metadata-updates).
```
Comment (markdboyd) [inline docs/external-contributions/external-contributions.md:27]:
```suggestion
GHRC has provided a [DMR++ Generation](https://github.com/ghrcdaac/dmrpp-generator#overview) workflow task. This task is meant to be used in conjunction with Cumulus' [Hyrax Metadata Updates workflow task](https://github.com/nasa/cumulus/tree/master/tasks/hyrax-metadata-updates#cumulushyrax-metadata-updates).
```

**Review by markdboyd — COMMENTED** [review]
Some comments

File: docs/external-contributions/external-contributions.md (hunk 1 above)
Comment (laurenfrederick) [reply docs/external-contributions/external-contributions.md]: yep, done

File: docs/external-contributions/external-contributions.md (hunk 2 above)
Comment (laurenfrederick) [reply docs/external-contributions/external-contributions.md]: I don't know that we do. They are tagged. I need to go through these docs and find a place to do that.

**Review by laurenfrederick — COMMENTED** [review]
@markdboyd Thank you for the feedback. Ready for rereview!

File: docs/external-contributions/external-contributions.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+---
+id: external-contributions
+title: External Contributions
+hide_title: false
+---
+
+# External Contributions
```
Comment (markdboyd) [inline docs/external-contributions/external-contributions.md:7]: If you use `hide_title: false`, then you don't need this

Comment (laurenfrederick) [reply docs/external-contributions/external-contributions.md]: Whoops, yep, thanks!

**Review by markdboyd — APPROVED** [review]

**Pull request merged**